client/BidBuy.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import styled from 'styled-components';
+import TopInfo from './TopInfo.js';
+import BidInfo from './BidInfo.js';
+import MiscInfo from './MiscInfo.js';
+
```

Comment:
Why all of your react component (client) files have .js extension instead of .jsx? Are you not configuring webpack for .jsx? I can see your code has jsx though.

Reply:
I've configured webpack to transpile both .js and .jsx files. There's no need to use a .jsx extension for React files. I will, however. be changing them to .jsx, since that's what the AirBnB linter wants.